store/copr/coprocessor.go

```diff
@@ -88,6 +89,7 @@ func (c *CopClient) Send(ctx context.Context, req *kv.Request, variables interfa
 	}
 	ctx = context.WithValue(ctx, tikv.TxnStartKey(), req.StartTs)
 	ctx = context.WithValue(ctx, util.RequestSourceKey, req.RequestSource)
```

Comment:
Seems missing the logic to update req.RequestSource from the new explicit request source

Reply:
The coprocessor is set here https://github.com/pingcap/tidb/pull/44770/files#diff-4df1b0c8387d228764f0114bb4bf76ffbb54f4bbb47b45756c139b339a2d8610R276-R309